app/views/drink_item/edit.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class = "menubody">
+
+  <h1>Edit <%= @drink_item.name %></h1>
+
+  <% if @drink_item %>
```

Comment:
It seems that this conditional is not needed. On line 3 your are using `@drink_item.name`, so if there were no `@drink_item` you would receive an error anyway, so the conditional is never needed.